module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/New-EntraBetaApplicationPasswordCredential.md

```diff
@@ -1,7 +1,6 @@
 ---
 title: New-EntraBetaApplicationPasswordCredential
-description: This article provides details on the Set-EntraBetaApplicationProxyConnector command.
-
+description: This article provides details on the New-EntraBetaApplicationPasswordCredential command.
 
```

Comment:
Replace:

## Description

The New-EntraBetaApplicationPasswordCredential cmdlet creates a password credential for an application in Microsoft Entra ID.

To:

## Description

The `New-EntraBetaApplicationPasswordCredential` cmdlet creates a password credential for an application in Microsoft Entra ID.

Reply:
Changes done.